wallet-new/src/Cardano/Wallet/API/V1/Types.hs

```diff
@@ -1503,6 +1894,185 @@ instance BuildableSafeGen Transaction where
 instance Buildable [Transaction] where
     build = bprint listJson
 
+instance Buildable [AddressLevel] where
+    build = bprint listJson
+
+-- | Source address and corresponding derivation path, for external wallet.
+data AddressAndPath = AddressAndPath
+    { aapSrcAddress :: !AddressAsBase58
+    -- ^ Source address in Base58-format.
+    , aapDerPath    :: ![AddressLevel]
```

Comment:
Actually, now that I see this. Is there a semantical difference between this and the `AddressPath` type defined above ?

Reply:
No. `AddressPath` contains 5 levels, but _currently_ we're using path from 2 levels only.